src/containers/EditProjectPage.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react'
+import { Header, Container } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import ProjectForm from '../components/ProjectForm'
+import { editProject } from '../actions/editProjectAction'
+import { fetchProjectInfo } from '../actions/getProjectInfoAction'
+import { setLastLocation } from '../actions/setLastLocationAction'
+
+export class EditProjectPage extends Component {
+  componentDidMount () {
+    // Set last path
+    const path = this.props.location.pathname
+    this.props.setLastLocation(path)
+
+    // Check if user is login
+    /*
+    if (
+      !this.props.cookies.get('WebsiteOne_session') &&
+      !this.props.loggedInUser.data
+    ) {
+      this.props.history.push({
+        pathname: '/login'
+      })
+    }
+    */
+    // Get Project info by slug
+    this.props.fetchProjectInfo(this.props.match.params.slug)
+  }
+
+  handleSubmit = async values => {
+    console.log(values)
+    const { title, description, status } = values
+    const { editProject, history, cookies, projectInfo } = this.props
+    await editProject({
+      id: projectInfo.id,
+      title,
+      description,
+      status,
+      cookies,
+      history
+    })
+    history.push(`/projects/${this.props.match.params.slug}`)
```

Comment:
I think we want this to be in a callback @ALau2088 so that it pushes the user only if the `axios` request resolved, no?